components/Search/DocumentResult.js

```diff
@@ -30,13 +22,25 @@ export default withT(({ t, node }) => {
     <ActionBar mode='feed' document={node.entity} />
   ) : null
 
+  const highlightStyle = useMemo(
+    () =>
+      css({
+        '& em': {
+          background: colorScheme.getCSSColor('overlayInverted'),
+          color: colorScheme.getCSSColor('textInverted'),
```

Comment:
Align with highlights in comments: 
```suggestion
          background: colorScheme.getCSSColor('alert'),
          color: colorScheme.getCSSColor('text'),
```
or change comments to align: https://github.com/orbiting/styleguide/pull/335/files#diff-e99c1fab5175405325c66e78406abfd011c4dbb0522ec4aa6f553b39b30a3ecdR125-R126

Reply:
The problem was that alert is not visible enough on dark mode. I'm going to change the alert colors a bit for them to be usable both as highlight Blocks as well as highlight text.